diff --git a/bin/setprj.sh b/bin/setprj.sh
index e6811afd..46e9dfb3 100644
--- a/bin/setprj.sh
+++ b/bin/setprj.sh
@@ -154,19 +154,3 @@ function gw {
         echo "No test.lxt, test.fst, test.vcd in the current folder"
     fi
 }
-
-# check that git hooks are present
-cp $JTFRAME/bin/post-merge $(git rev-parse --git-path hooks)/post-merge
-
-# Recompiles jtframe quietly after each commit
-cd $JTFRAME
-JTFRAME_POSTCOMMIT=$(git rev-parse --git-path hooks)/post-commit
-cat > $JTFRAME_POSTCOMMIT <<EOF
-#!/bin/bash
-jtframe > /dev/null
-if [ $(git branch --no-color --show-current) = master ]; then
-    # automatically push changes to master branch
-    git push
-fi
-EOF
-chmod +x $JTFRAME_POSTCOMMIT
